### ISSUE TITLE
What is the difference between t2 and t2_ft?

### ISSUE THREAD

**Issue by 343gltysprk** [feifeiobama/OrthogonalDet#3]
In the ft task, are you using previous known classes as validation set?

**Comment by feifeiobama** [comment]
The difference is in the curation of the training data: t2 uses only the new classes for training, while t2_ft uses all previously known classes which simulates the replay strategy.
